src/Utils.php

```diff
@@ -275,6 +275,28 @@ public static function buildPropsArray($items, $schema, $isData = FALSE) {
       }
     }
 
+    if ($isData) {
```

Comment:
Moved this logic to Utils.php as requested here https://gitlab.com/pegadigital/bolt/design-system/-/merge_requests/172/diffs#abc6908e4ce44dffaf341cb178f23e9d085294ce_236_247